frontend/src/App.tsx

```diff
@@ -258,6 +259,8 @@ function App() {
               <main className={classes.content}>
                 <div className={classes.toolbar} />
                 <Container maxWidth="lg">
+                  <NavigationTabs/>
+                  <Divider />
```

Comment:
This divider should be shown only if the tabs are in use (otherwise it will look out of place).
![Screenshot from 2020-11-04 12-43-49](https://user-images.githubusercontent.com/1029635/98116283-fbea4600-1e9f-11eb-943a-2f5e940171ab.png)